### ISSUE TITLE
AutoCompleteComboBox not show items

### ISSUE THREAD

**Issue by SimonDarksideJ** [Unity-UI-Extensions/com.unity.uiextensions#343]
<sup>Issue created by Rodrigo Meireles as [Bitbucket Issue #&#8203;204](https://bitbucket.org/UnityUIExtensions/unity-ui-extensions/issues/204) on 2018.01.15 17:10.</sup>
Hi, I add items with 

myautocompletecombobox.AvailableOptions.Add ("Test1");
myautocompletecombobox.AvailableOptions.Add ("Test2");
myautocompletecombobox.AvailableOptions.Add ("Test3");
myautocompletecombobox.AvailableOptions.Add ("Test4");
myautocompletecombobox.AvailableOptions.Add ("Test5");

on start of my script but not show when play my app

on inspector show the items but not on my app running

my unity is 2017.2

**Comment by SimonDarksideJ** [comment]
<sup>On [2018.01.18 16:54](https://bitbucket.org/UnityUIExtensions/unity-ui-extensions/issues/204#comment-42722750), @SimonDarksideJ commented:</sup>
OK, so you are programatically adding items while running.  I admit, I haven't tested that use case, so I'll need to look in to in.

**Comment by SimonDarksideJ** [comment]
<sup>On [2020.08.26 22:58](https://bitbucket.org/UnityUIExtensions/unity-ui-extensions/issues/204#comment-58602382), JBKacerovsky commented:</sup>
I encountered a similar issue using Combobox. I was able to solve it by including the following lines in the Combobox script to set the available options while running and to update thee displayed list.

```
        public void SetAvailableOptions(List<string> newOptions)
        {
            AvailableOptions = newOptions;
            RebuildPanel();
        }
```

then I simply call myComboBox.SetAvailableOptions\(ListOfNewOptionStings\) from the script. 

 Something similar should work for autocompletecombobox, I think.

**Comment by SimonDarksideJ** [comment]
<sup>On [2020.08.27 17:51](https://bitbucket.org/UnityUIExtensions/unity-ui-extensions/issues/204#comment-58612111), @SimonDarksideJ commented:</sup>
Great resolution, Will update those controls ASAP.

**Comment by SimonDarksideJ** [comment]
<sup>On [2020.08.27 18:20](https://bitbucket.org/UnityUIExtensions/unity-ui-extensions/issues/204#comment-58612344), @SimonDarksideJ commented:</sup>
Updated DropDown and Autocomplete controls based on feedback in #204

Resolves #204


Referenced by 3429d37f144b

**Comment by SimonDarksideJ** [comment]
<sup>On 2020.08.27 18:20 @SimonDarksideJ modified issue:</sup>
**status** changed `new` → `resolved`
